src/Trixi.jl

```diff
@@ -160,6 +160,7 @@ export initial_condition_lid_driven_cavity, boundary_condition_lid_driven_cavity
 export initial_condition_couette_steady, initial_condition_couette_unsteady, boundary_condition_couette
 export initial_condition_gauss_wall
 export initial_condition_monopole, boundary_condition_monopole
+export initial_condition_rti, source_terms_rti
```

Comment:
I have this in a `DGMulti` elixir, but was planning on committing it in a separate PR. What if I were to add the elixir to this PR and keep the initial condition in `equations/`?

Reply:
Hm. Really, I don't know if in this case we'd wouldn't still prefer to have it twice in the elixirs. What about you, @trixi-framework/principal-developers?

Reply:
I'm fine with just moving the initial condition into the elixirs too.

Reply:
:+1

Reply:
I agree. In this case, it will be nice to have these in the elixirs, even if we have multiple elixirs using this IC. For me, the main reason for this is that it is a classical setup many people looking at Trixi.jl will know. Thus, it is a good starting point for them to create their own setups. Including the ICs in the elixir makes this much easier.

Reply:
OK. I'll add that along to this PR if @sloede and @ranocha don't mind.

Reply:
Sure.